test/unit_tests/view_model/InputBarViewModelSpec.js

```diff
@@ -22,6 +22,7 @@
 // grunt test_init && grunt test_run:view_model/InputBarViewModel
 
 describe('z.viewModel.content.InputBarViewModel', () => {
+  /*
```

Comment:
What's this? If we remove the tests we can remove it.

Reply:
I wanted to keep this init code somewhere for the moment we actually write some tests.
Unfortunatly because the only test there was a method that was removed, this code is dead and eslint complains about it.

My point here, is that I want people who will want to write test in the future to be able to write them quickly without having to think about how to instantiate the ViewModel.